NBitcoin/TransactionBuilder.cs

```diff
@@ -1377,7 +1387,12 @@ bool IsEconomical(Coin c)
 				var input = ctx.Transaction.Inputs.FirstOrDefault(i => i.PrevOut == coin.Outpoint);
 				if(input == null)
 					input = ctx.Transaction.Inputs.Add(coin.Outpoint);
-				if(_LockTime != null && !ctx.NonFinalSequenceSet)
+
+				if(OptInRBF)

```

Comment:
Here is where I am not 100% sure it is correct.

Reply:
It is not correct. Inverse the if and the if/else.
You are doing `if A => RBF elseif B => RBF+STUFF`

So if A is true, we are missing `STUFF`.
So you should do

`if B => RBF+STUFF elseif A =>RBF`

(Note that setting sequence to 0 effecitively put RBF on)

Reply:
Or remove the if else.

`if A => RBF; if B => RBF + STUFF`

Reply:
> (Note that setting sequence to 0 effectively activate RBF)
F*ck! You're right.